Userland/Libraries/LibVideo/VP9/Parser.cpp

```diff
@@ -865,8 +869,15 @@ DecoderErrorOr<void> Parser::decode_tiles(FrameContext& frame_context)
     //        then run through each column of tiles in top to bottom order afterward. Each column can be sent to a worker thread
     //        for execution. Each worker thread will want to create a set of above contexts sized to its tile width, then provide
     //        those to each tile as it decodes them.
-    for (auto tile_row = 0; tile_row < tile_rows; tile_row++) {
-        for (auto tile_col = 0; tile_col < tile_cols; tile_col++) {
+    Vector<Vector<TileContext, 1>, 4> tile_workloads;
+    DECODER_TRY_ALLOC(tile_workloads.try_ensure_capacity(tile_cols));
+    for (auto tile_col = 0u; tile_col < tile_cols; tile_col++) {
+        tile_workloads.append({});
+        tile_workloads[tile_col].ensure_capacity(tile_rows);
```

Comment:
Ayy you sneaked this in :^) shouldn't this be `try_ensure_capacity`?

Reply:
Oops, yeah, not awake enough today I suppose.